utopia-remix/app/services/fgaService.server.ts

```diff
@@ -0,0 +1,103 @@
+import { CredentialsMethod, OpenFgaClient } from '@openfga/sdk'
+import { AccessLevel } from '../types'
+
+export const fgaClient = new OpenFgaClient({
+  apiScheme: 'https',
+  apiHost: 'api.us1.fga.dev',
+  storeId: process.env.FGA_STORE_ID!,
```

Comment:
I believe there's some handling for environment variables in this project, so that it's possible to make them required. Adding `!` on something in TypeScript just makes it think the type isn't nullable even if the value is null.